src/main/resources/META-INF/plugin-github.xml

```diff
@@ -0,0 +1,6 @@
+<idea-plugin>
+    <extensions defaultExtensionNs="com.intellij">
+        <applicationService serviceImplementation="com.redhat.devtools.intellij.knative.git.GitHandler"/>
```

Comment:
I don't get it, why do you declare them as service if you don't use them as such ?

Reply:
No valid reason, just a lack of alternatives. I don't really need anything as I check if the gh plugin is installed and I disable the action myself. So I used services as they are loaded on demand so if there is no gh plugin they are skipped. Which tag should i use?

Reply:
Can't you simply have an empty file ?

Reply:
:expressionless: ... done